configure.ac

```diff
@@ -145,6 +146,20 @@ CPPFLAGS="-D_REENTRANT -D_THREAD_SAFE $CPPFLAGS"
 # For host type checks
 AC_CANONICAL_HOST
 
+#For a working getsockopt() optname=SO_ERROR
+dnl AC_MSG_CHECKING([for getsockopt optname SO_ERROR)])
```

Comment:
On Sat, 2016-01-16 at 05:17 -0800, Luca Boccassi wrote:
> In configure.ac:
> > @@ -145,6 +146,20 @@ CPPFLAGS="-D_REENTRANT -D_THREAD_SAFE $CPPFLAGS"
> >  # For host type checks
> >  AC_CANONICAL_HOST
> >  
> > +#For a working getsockopt() optname=SO_ERROR
> > +dnl AC_MSG_CHECKING([for getsockopt optname SO_ERROR)])
> On the Debian bug discussion [1], @sthibaul mentioned that this sockopt is in
> fact available on Hurd. Also doesn't look like HAVE_SOCKLEN_T is used. Any
> reason to keep this section?
> [1] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=799235
> —
> Reply to this email directly or view it on GitHub.

The dnl stuff is just commented out code, updated patch attached.
Sorry for not replying earlier, I did not get the bug mails since I'm not the
bug submitter.